cms/templates/cms/program_page.html

```diff
@@ -130,7 +130,7 @@ <h3 class="title">
           <p>
             Courses delivered on
           </p>
-          <img src="/static/images/edx_logo.png" alt="edX" />
+          <img src="{% static 'images/edx_logo.png' %}" alt="edX" />
```

Comment:
i suppose this is the image

<img width="1246" alt="screen shot 2017-03-15 at 4 07 35 pm" src="https://cloud.githubusercontent.com/assets/10431250/23945759/984e5cae-0999-11e7-81b1-4157acfaabb7.png">